app/views/FirstContactEmailView.scala.html

```diff
@@ -24,9 +24,11 @@
     govukButton: GovukButton
 )
 
-@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+@(form: Form[_], mode: Mode, name: String = "[SOME BLOKE]")(implicit request: Request[_], messages: Messages)
```

Comment:
the name page is not in the skeleton so DAC6-2733 must be complete before adding this dynamic text